src/components/bottom-bar/BottomBar.tsx

```diff
@@ -178,33 +225,46 @@ const BottomBar = () => {
   )
 
   return !hideBottomBar ? (
-    <SafeAreaView style={bottomBarStyle} edges={['bottom']}>
-      <FeedButton
-        isActive={currentRoute === FEED_PAGE}
-        isDarkMode={isDarkMode}
-        onClick={onClick(goToFeed, FEED_PAGE)}
-      />
-      <TrendingButton
-        isActive={currentRoute === TRENDING_PAGE}
-        isDarkMode={isDarkMode}
-        onClick={onClick(goToTrending, TRENDING_PAGE)}
-      />
-      <ExploreButton
-        isActive={currentRoute === EXPLORE_PAGE}
-        isDarkMode={isDarkMode}
-        onClick={onClick(goToExplore, EXPLORE_PAGE)}
-      />
-      <FavoritesButton
-        isActive={currentRoute === FAVORITES_PAGE}
-        isDarkMode={isDarkMode}
-        onClick={onClick(goToFavorites, FAVORITES_PAGE)}
-      />
-      <ProfileButton
-        isActive={currentRoute === userProfilePage}
-        isDarkMode={isDarkMode}
-        onClick={onClick(goToProfile, userProfilePage)}
-      />
-    </SafeAreaView>
+    <Animated.View
```

Comment:
Gonna be a fun merge with https://github.com/AudiusProject/audius-mobile-client/pull/155 but I'll get it figured out

Reply:
Thanks! sorry, i can do the rebase too if you think it'll be easier that way. lmk if you want to sync about it real time